auth0_flutter/analysis_options.yaml

```diff
@@ -52,7 +52,7 @@ linter:
     join_return_with_assignment: true
     leading_newlines_in_multiline_strings: true
     library_private_types_in_public_api: true
-    lines_longer_than_80_chars -> packages only: true
```

Comment:
Flutter CLI doesn't know what `packages only` does any more, and I can't find much on the web about it.

Reply:
That's a comment by me, which I forgot to remove when I uncommented the line. My bad 🤦🏼‍♀️

Reply:
All good, I had thought it was some kind of special hidden syntax we were using 😄